src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -392,7 +510,7 @@ private void printUsage(final PrintStream destinationStream, final Set<Class<?>>
     /**
      * @return A display name to be used for the tool who's class is {@code clazz}.
      */
-    protected String getDisplayNameForToolClass(Class<?> clazz) {
+    protected static String getDisplayNameForToolClass(Class<?> clazz) {
```

Comment:
@jonn-smith The reason several of these methods were changed to be instance methods was to allow toolkits to override the tool name resolution behavior. If these are reverted to static they can't be overridden by subclasses.

Reply:
As @cmnbroad said, non static (here and above)

Reply:
Sounds good - I'll pull out the static.